.gitlab-ci.yml

```diff
@@ -39,7 +39,7 @@ before_script:
     - go get -u github.com/axw/gocov/gocov golang.org/x/tools/cmd/cover github.com/fzipp/gocyclo
 
 test:prepare_acceptance:
-  image: docker
+  image: docker:19.03.5-dind
```

Comment:
This one should not have been necessary, though. But the Pipeline passed so I guess it is not critical to clean up.